it/apps/src/main/content/jcr_root/apps/forms-core-components-it/clientlibs/clientlib-it-base/styles/site.css

```diff
@@ -1895,4 +1895,8 @@ pre.prettyprint .fun{color:#d7373f}
 .cmp-progressbar .cmp-progressbar__label--completed:after{content:'%'}
 .cmp-progressbar .cmp-progressbar__label--remaining{float:right;color:#2e6ca3;padding-right:.3125rem}
 .cmp-progressbar .cmp-progressbar__label--remaining:after{content:'%'}
-.cmp-progressbar .cmp-progressbar__bar{display:block;height:100%;font-size:0;background-color:#2e6ca3}
\ No newline at end of file
+.cmp-progressbar .cmp-progressbar__bar{display:block;height:100%;font-size:0;background-color:#2e6ca3}
+
+
+.ReplaceComponentDialog-components {padding: 0;}
```

Comment:
lets follow bem notation. css for this should be in the component rather than here.